app/src/main/java/org/oppia/android/app/profile/AdminAuthActivityPresenter.kt

```diff
@@ -107,6 +108,7 @@ class AdminAuthActivityPresenter @Inject constructor(
           context.resources.getString(R.string.admin_auth_admin_controls_sub)
       }
       AdminAuthEnum.PROFILE_ADD_PROFILE.value -> {
+        activity.title = context.getString(R.string.admin_auth_activity_add_profiles_title)
         binding?.adminAuthToolbar?.title = context.resources.getString(R.string.add_profile_title)
```

Comment:
Ditto here. @rt4914 are the toolbar & activity titles different in this activity?

Reply:
Yes @BenHenning they are different.
If the learner clicks on `Add Profiles` the UI is different in terms of toolbar title and the subtitle. The toolbar in this case shows `Add Profiles`. Now given that this is the same label for actual `AddProfileActivity` we are using label for talkback as `Authorise to add profiles` so that learner can easily understand.

If the learner clicks on `Admin Controls` the UI toolbar shows `Administrator Controls` and the label is `Authorise to access Administrator Controls`.

See below screenshots for more clarity.
<img src="https://user-images.githubusercontent.com/9396084/111470546-fd7d2200-874d-11eb-84d0-89b8f3a2ef8f.png" width="250" />
<img src="https://user-images.githubusercontent.com/9396084/111470558-ff46e580-874d-11eb-8f60-e451cbb04118.png" width="250" />

Reply:
Thanks for the added context.